lib/ansible/modules/remote_management/ucs/ucs_timezone.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_timezone
+
+short_description: Configures timezone on Cisco UCS Manager
+
+version_added: "2.6"
+
+description:
+    - Configures time zone on Cisco UCS Manager.
+    - Examples can be used with the UCS Platform Emulator U(https://communities.cisco.com/ucspe).
+
+options:
+    state:
+        description:
+            - If C(present), will verify timezone is set (persent) and will set (create) if needed.
+            - If C(absent), will verify timezone is unset (absent) and will unset (delete) if needed.
+        choices: [present, absent]
+        default: absent
+        required: true
```

Comment:
We also generally don't require state with the default of present.

Reply:
In fact Ansible is now adding tests to make this impossible in the future.

Reply:
removed required